docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,241 @@
+
+```typescript
+    //   content: hi world
+    // positions: 01234567
+
+    let props1 = sharedString.getPropertiesAtPosition(1);
+    let props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = {}
+    // props5 = {}
+
+    // set property called weight on positions 0 and 1
+    sharedString.annotateRange(0, 2, { weight: 5 });
+    props1 = sharedString.getPropertiesAtPosition(1);
+    props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = { weight: 5 }
+    // props5 = {}
+
+    // set property called decoration on all positions
+    sharedString.annotateRange(
+        0,
+        sharedString.getLength(),
+        { decoration: "underline" });
+    props1 = sharedString.getPropertiesAtPosition(1);
+    props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = { weight: 5, decoration: "underline" }
+    // props5 = { decoration: "underline" }
+
+    // remove property called weight on all positions
+    sharedString.annotateRange(
+        0,
+        sharedString.getLength(),
+        { weight: null });
+    props1 = sharedString.getPropertiesAtPosition(1);
+    props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = { decoration: "underline" }
+    // props5 = { decoration: "underline" }
+```
+
+Whenever an operation is performed on a sequence a *sequenceDelta* event will be raised. This even provides the ranges
+affected by the operation, the type of the operation, and the properties that were changes by the operation.
+
+## How Collaboration Works
+
+Like other data structures the sequences are eventually consistent which means all collaborators will end up in the same
+final state, however, the intermediate states seen by each collaborator may not be seen by other collaborators. These
+intermediate states occur when two or more collaborators modify the same position in the sequence which results in a
+conflict.
+
+The basic strategy for insert conflict resolution in the sequence is to merge *far*. This strategy depends on a
+fundamental property of the Fluid Framework, which is guaranteed ordering. So, if two or more collaborators perform an
+operation on a sequence, the operations will be given an ordering and all clients will see those operations in the same
+order. What this means for the merge *far* strategy for resolving conflicting inserts is that the first operation will
+be placed in the conflicting position when it is received. When the next insert with the same position arrives and is
+applied it will be placed at the specified position and the previous inserts content position will be increased by the
+length of the incoming content pushing is farther towards the length of the sequence. This is what we call merging
+*far*.
+
+Like insert the strategies for remove and annotate also rely on guaranteed ordering. For remove and annotate only
+content visible to the collaborator creating the operation will be modified, any content ordered after the won't be.
```

Comment:
I do not think this paragraph answer the question of what happens if user A inserts at pos = 2, while user B deletes 0..4 range. Or at least - most people learn by samples, and it would be great to have that sample right after that paragraph.